test/ngRoute/routeSpec.js

```diff
@@ -1677,62 +1678,380 @@ describe('$route', function() {
   });
 
 
-  describe('reloadOnSearch', function() {
-    it('should reload a route when reloadOnSearch is enabled and .search() changes', function() {
-      var reloaded = jasmine.createSpy('route reload');
+  describe('reloadOnUrl', function() {
+    it('should reload a route when `reloadOnUrl` is enabled and `.url()` changes', function() {
```

Comment:
The wording is inconsistent in the changes - mostly it's `enabled` but further below it's `true` (I prefer the latter).

Reply:
Made consistent.